src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -172,9 +203,32 @@ protected override void Write(LogEventInfo logEvent)
                     return;
                 }
 
+                // bodge to try to get the exception message in English
+                // https://stackoverflow.com/questions/209133/exception-messages-in-english
+                // There may still be some localization but this is better than nothing.
+                string message = string.Empty;
+
+                // only try to use the exeception message to fingerprint if there's no inner
+                // exception and the message is short, otherwise we're in danger of getting a
+                // stacktrace which will break the grouping
+                if (logEvent.Exception != null && logEvent.Exception.InnerException == null)
+                {
+                    var t = new Thread(() => {
+                            message = logEvent.Exception?.Message;
+                        });
+                    t.CurrentCulture = CultureInfo.InvariantCulture;
+                    t.CurrentUICulture = CultureInfo.InvariantCulture;
+                    t.Start();
+                    t.Join();
+                }
+
+                Console.WriteLine($"Sentry fingerprint message {message}");
```

Comment:
Is this console write needed?

Reply:
Sorry, thought I'd got rid of all of those